libs/@hashintel/ds-components/figma.config.json

```diff
@@ -0,0 +1,7 @@
+{
+  "codeConnect": {
+    "include": ["src/components/**/*.{tsx,jsx}"],
+    "label": "React",
+    "interactiveSetupFigmaFileUrl": "https://www.figma.com/design/WmnosvOvi4Blw0HK0jh1mG/Design-System?m=auto&t=d2L1QyposDsmx9sl-6"
```

Comment:
Not sure what is `t` here but it is not my personal token.

This was added by the setup.